pkg/controller/knativeeventing/knativeeventing_controller.go

```diff
@@ -274,6 +275,16 @@ func addSCCforSpecialClusterRoles(u *unstructured.Unstructured) error {
 	return nil
 }
 
+func setSideEffects(u *unstructured.Unstructured) error {
```

Comment:
this will be fixed upstream - I hope.

@aliok if not ... this might be a bit of "specfic code" for us, required ... since Openshift 4.3 does dry-runs

Reply:
@matzew thanks for letting me know. not sure why we need this, but going to have it in the new operator.

Reply:
https://issues.redhat.com/browse/SRVKE-283 is the bug